msg/HelperROT.msg

```diff
@@ -0,0 +1,3 @@
+# Unit: -126 to 126 ROT AIS scale corresponds to -708 to 708 degrees per min
```

Comment:
To me it is a bit unclear what this is. Would this be correct?

```
# Rate of turn
# Unit: degrees per min, increases CW
# Range: [-708, 708)
# To convert to ROT AIS scale [-126, 126): 71 / 15 * sqrt(rot)
```

Reply:
Also you can't square root a negative number

Reply:
Yes that would be correct. I added a link, in the PR description, to the page where I got the info for ROT.
Yes we would need to handle the incoming ROT AIS data carefully to preserve the negative sign when we convert it from [-126, 126) to [-708, 708) for use in our code. This is mentioned in the page I linked in the description also.

Just to clarify, we wont be taking the square root of anything, we will be receiving a value that we need to divide by (71/15), then square, and then preserve the negative sign if the original value was negative.

I realize having the conversion in reverse in the msg file is sort of confusing, so I can change that. As of right now ROT isn't used for anything in the codebase, so I haven't implemented this conversion function anywhere yet.